warehouse/query-core/src/main/java/datawave/query/jexl/LiteralRange.java

```diff
@@ -4,17 +4,21 @@
 import org.apache.commons.lang.builder.CompareToBuilder;
 import org.apache.commons.lang.builder.EqualsBuilder;
 import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Preconditions;
 
 /**
  * The values of T should already be normalized
  *
- *
- *
  * @param <T>
  *            type of the range
  */
 public class LiteralRange<T extends Comparable<T>> implements Comparable<LiteralRange<T>> {
 
+    private static final Logger log = LoggerFactory.getLogger(LiteralRange.class);
```

Comment:
As far as I can tell, this Logger instance isn't used anywhere.

Reply:
removed in next commit